### PR TITLE
refactor(www): Turn featured-sites to function component and simplify media queries.

### PR DESCRIPTION
**Pull request by tesseralis**
## Description

Turn featured-sites to function component and simplify media queries.

### CODE REVIEW

File: www/src/views/showcase/featured-sites.js [hunk 1]
```diff
@@ -22,16 +18,9 @@ const featuredSitesCard = {
   flexDirection: `column`,
   flexGrow: 0,
   flexShrink: 0,
-  width: 320,
+  width: [320, null, null, null, 360, 400],
```
Comment (tesseralis) [inline www/src/views/showcase/featured-sites.js:21]: hmm… the margins are wrong here.

File: www/src/views/showcase/featured-sites.js [hunk 2]
```diff
@@ -58,244 +47,219 @@ const GradientOverlay = () => {
...
+            },
+          }}
+          onClick={setFilterToFeatured}
+        >
+          <span className="title">View all</span>
+          {` `}
+          <ArrowForwardIcon sx={{ verticalAlign: `sub` }} />
+        </a>
+        <div
+          css={{
```
Comment (abhishekjakhar) [inline www/src/views/showcase/featured-sites.js:106]:
```suggestion
          sx={{
```

File: www/src/views/showcase/featured-sites.js [hunk 3]
```diff
@@ -58,244 +47,219 @@ const GradientOverlay = () => {
...
             alignItems: `center`,
-            flexWrap: `wrap`,
+            display: `flex`,
+            marginLeft: `auto`,
```
Comment (abhishekjakhar) [inline www/src/views/showcase/featured-sites.js:109]:
```suggestion
            ml: `auto`,
```

File: www/src/views/showcase/featured-sites.js [hunk 4]
```diff
@@ -58,244 +47,219 @@ const GradientOverlay = () => {
...
+            target="_blank"
+            rel="noopener noreferrer"
+            variant="small"
+            icon={<ArrowForwardIcon />}
           >
+            Submit your Site
+          </Button>
+        </div>
+      </div>
+      <div css={{ position: `relative` }}>
```
Comment (abhishekjakhar) [inline www/src/views/showcase/featured-sites.js:134]:
```suggestion
      <div sx={{ position: `relative` }}>
```

File: www/src/views/showcase/featured-sites.js [hunk 5]
```diff
@@ -58,244 +47,219 @@ const GradientOverlay = () => {
...
-                href="#showcase"
+            </div>
+          ))}
+          <div sx={{ display: `flex` }}>
+            <a
+              href="#showcase"
+              sx={{
+                backgroundColor: `card.background`,
+                borderRadius: 1,
+                marginRight: t => `${t.space[6]} !important`,
```
Comment (abhishekjakhar) [inline www/src/views/showcase/featured-sites.js:211]:
```suggestion
                mr: t => `${t.space[6]} !important`,
```
can we also see if !important is necessary here, or can we remove it?

File: www/src/views/showcase/featured-sites.js [hunk 6]
```diff
@@ -22,16 +18,9 @@ const featuredSitesCard = {
   flexDirection: `column`,
   flexGrow: 0,
   flexShrink: 0,
-  width: 320,
+  width: [320, null, null, null, 360, 400],
   marginBottom: 9,
```
Comment (abhishekjakhar) [inline www/src/views/showcase/featured-sites.js:22]:
```suggestion
  mb: 9,
```

File: www/src/views/showcase/featured-sites.js [hunk 7]
```diff
@@ -22,16 +18,9 @@ const featuredSitesCard = {
...
-  marginRight: 6,
-  [mediaQueries.xl]: {
-    width: 360,
-    marginRight: 8,
-  },
-  [mediaQueries.xxl]: {
-    width: 400,
-  },
+  marginRight: [6, null, null, null, 8],
```
Comment (abhishekjakhar) [inline www/src/views/showcase/featured-sites.js:23]:
```suggestion
  mr: [6, null, null, null, 8],
```

**Pull request closed without merging**